src/main/java/bisq/core/proto/CoreProtoResolver.java

```diff
@@ -18,30 +18,7 @@
 package bisq.core.proto;
 
 import bisq.core.payment.AccountAgeWitness;
-import bisq.core.payment.payload.AliPayAccountPayload;
-import bisq.core.payment.payload.CashAppAccountPayload;
-import bisq.core.payment.payload.CashDepositAccountPayload;
-import bisq.core.payment.payload.ChaseQuickPayAccountPayload;
-import bisq.core.payment.payload.ClearXchangeAccountPayload;
-import bisq.core.payment.payload.CryptoCurrencyAccountPayload;
-import bisq.core.payment.payload.FasterPaymentsAccountPayload;
-import bisq.core.payment.payload.InteracETransferAccountPayload;
-import bisq.core.payment.payload.MoneyBeamAccountPayload;
-import bisq.core.payment.payload.NationalBankAccountPayload;
-import bisq.core.payment.payload.OKPayAccountPayload;
-import bisq.core.payment.payload.PaymentAccountPayload;
-import bisq.core.payment.payload.PerfectMoneyAccountPayload;
-import bisq.core.payment.payload.PopmoneyAccountPayload;
-import bisq.core.payment.payload.RevolutAccountPayload;
-import bisq.core.payment.payload.SameBankAccountPayload;
-import bisq.core.payment.payload.SepaAccountPayload;
-import bisq.core.payment.payload.SepaInstantAccountPayload;
-import bisq.core.payment.payload.SpecificBanksAccountPayload;
-import bisq.core.payment.payload.SwishAccountPayload;
-import bisq.core.payment.payload.USPostalMoneyOrderAccountPayload;
-import bisq.core.payment.payload.UpholdAccountPayload;
-import bisq.core.payment.payload.VenmoAccountPayload;
-import bisq.core.payment.payload.WesternUnionAccountPayload;
+import bisq.core.payment.payload.*;
```

Comment:
Please don't collapse imports like this. If you're using a recent version of IntelliJ IDEA, you should pick up the correct import organization settings. If not, then you just need to be careful to follow conventions.

Reply:
@cbeams @ripcurlx ,

Just double checked, it became this style when I 'auto import'.

Tried again with fallback to a long list of dependency and then manually add the wechatAccountPayload import, but once I commit with option 'optimize import' then it would become * again.(refer my latest commit)

I think the reason is because this class really used all Payload.java in payload folder?